src/AgGrid.fs

```diff
@@ -7,6 +7,7 @@ open Fable.Core.JsInterop
 open Feliz
 
 let agGrid : obj = import "AgGridReact" "ag-grid-react"
+let allModules : obj = importAll "ag-grid-community"
```

Comment:
`allModules` seems like a weird name for the public API. What is it and what stuff does it expose?